tests/unit/swagger/definitionsGeneration/metadata.spec.ts

```diff
@@ -503,6 +503,22 @@ describe('Metadata generation', () => {
     });
   });
 
+  describe('HiddenControllerGenerator', () => {
+    const parameterMetadata = new MetadataGenerator('./tests/fixtures/controllers/hiddenController.ts').Generate();
+    const controller = parameterMetadata.controllers[0];
+
+    it('should generate hidden methods', () => {
```

Comment:
I'm a little confused by the message `'should generate hidden methods'`. Do you mean `'should generate metadata for the hidden methods even though it won't generate swagger definitions'`?

Speaking of which, can you add a unit test that confirms that the swagger output does not contain the methods of the hidden controller?

I'm asking because I believe that is the primary goal of the `@Hidden` decorator, right?

You'll need to do those in two places:
* For Swagger (i.e. OpenAPI v2) `tests\unit\swagger\definitionsGeneration\definitions.spec.ts`
* For OpenAPI 3: `tests\unit\swagger\schemaDetails3.spec.ts`

Other than that, this is a great PR. Thank you for adding to the readme. Well done! :)

Reply:
Sure thing, I can make those changes.  The test message was just a victim of copy & paste from the existing `@Hidden` tests.

Reply:
> For Swagger (i.e. OpenAPI v2) `tests\unit\swagger\definitionsGeneration\definitions.spec.ts`

Can you provide a little more guidance on this? I don't see any `paths` tests in that file.

Reply:
You're correct. I just checked and there are not any path tests in that file. So I'm not sure where it's best to put those tests, but we do need to make sure that this decorator works for both Swagger 2.0 and Open API 3.0